lib/grpc/client/resolver/xds.ex

```diff
@@ -1,4 +1,5 @@
 defmodule GRPC.Client.Resolver.XDS do
+  @moduledoc false
```

Comment:
This is yet another sign to remove XDS mentions from the docs and API

Reply:
Since the scope here is documentation, I will remove or implement this later in another PR. I have already removed it from the documentation for now.